dashboard/app/controllers/scripts_controller.rb

```diff
@@ -260,13 +261,17 @@ def get_rollup_resources
     end
   end
 
-  private def get_unit
+  private def get_unit_by_name
     unit_id = params[:id]
```

Comment:
How much work would it be to update the parameter used here to be called `name`? It was already a little weird to use the name `id` when it actually represented "either name or id", but it's even more confusing now.

Reply:
I agree that this is confusing! This would be a relatively large change, affecting the following routes: https://github.com/code-dot-org/code-dot-org/blob/710ce4a10c6ec4d741b9573dbdf75ec571440cb5/dashboard/config/routes.rb#L453-L513

given that I am already 3 layers deep on cleanup I'd like to pass on adding anything this big to my queue at this time.